Sources/SwiftParser/CompilerFiles.swift

```diff
@@ -0,0 +1,102 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if compiler(>=6)
+@_spi(RawSyntax) @_spi(ExperimentalLanguageFeatures) internal import SwiftSyntax
+#else
+@_spi(RawSyntax) @_spi(ExperimentalLanguageFeatures) import SwiftSyntax
+#endif
```

Comment:
Why do you need `internal import` here?

Reply:
Not sure I understand your question. Every swift files in `SwiftParser` have `internal import` or `public import` depending on the APIs declared in the file.

Reply:
Ooops, I thought that we didn’t use scoped imports at all in swift-syntax. Sorry.